specs/analytics_v1_pageviews.yaml

```diff
@@ -0,0 +1,295 @@
+          type: string
+          required: true
+        - name: end
+          in: path
+          description: End timestamp in YYYYMMDDHH format
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The list of values
+          schema:
+            $ref: '#/definitions/project'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-request-handler:
+        - get_from_backend:
+          request:
+            uri: /{domain}/sys/pageviews/per-project/{project}/{access}/{agent}/{granularity}/{start}/{end}
+      x-monitor: false
+
+  /{module:pageviews}/top/{project}/{access}/{year}/{month}/{day}:
+    get:
+      tags:
+        - Pageviews data
+        - Top articles
+      description: >
+        List pageviews for a given project, over a given time period.
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: project
+          in: path
+          description: Name of the project [all-projects|project-name]
+          type: string
+          required: true
+        - name: access
+          in: path
+          description: Type of access method [all-access|desktop|mobile-app|mobile-web]
+          type: string
+          required: true
+        - name: year
+          in: path
+          description: The year for which to retrieve top articles [all-years|year]
+          type: string
+          required: true
+        - name: month
+          in: path
+          description: The month for which to retrieve top articles [all-months|month]
+          type: string
+          required: true
+        - name: day
+          in: path
+          description: The day for which to retrieve top articles [all-days|day]
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The list of top articles in the project
+          schema:
+            $ref: '#/definitions/tops'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-request-handler:
+        - get_from_backend:
+          request:
+            uri:  /{domain}/sys/pageviews/top/{project}/{access}/{year}/{month}/{day}
+      x-monitor: false
+
+
+definitions:
+  # A https://tools.ietf.org/html/draft-nottingham-http-problem
+  problem:
+    required:
+      - type
+    properties:
+      type:
+        type: string
+      title:
+        type: string
+      detail:
+        type: string
+      instance:
+        type: string
+
+  listing:
+    description: The result format for listings
+    required:
+      - items
+    properties:
+      items:
+        type: array
+        items:
+          type: string
+
+  article:
+    required:
+      - items
+    properties:
+      items:
+        type: array
+        properties:
```

Comment:
This is a invalid swagger. Should be 
```yaml
properties:
  items:
    type: array
    items:
      properties:
        ...
```
Alternatively you could put the description of an individual object to a separate definition and use `$ref` stanza like [here](https://github.com/wikimedia/restbase/blob/master/specs/mediawiki/v1/content.yaml#L1177)

This comment applies to many places in this file, to check that you've fixed all of them - start the service and load [docs](http://localhost:7231/en.wikipedia.org/v1/?doc#/) - if it loads - all is fine.